scan/lib/scan/test_command_generator.rb

```diff
@@ -95,6 +95,14 @@ def xcodebuild_log_path
         return File.join(containing, file_name)
       end
 
+      # Generate destination parameters
```

Comment:
Ah, I wasn't aware that `xcodebuild` supports passing multiple destinations. I initially thought that we'd call it multiple times. However I'm sure it's faster like this 👍